index.bs

```diff
@@ -33,10 +33,14 @@ urlPrefix: https://tc39.es/ecma262/; spec: ECMA-262
     type: dfn
         text: element size; url: table-the-typedarray-constructors
         text: element type; url: table-the-typedarray-constructors
+        text: view constructor; url: table-the-typedarray-constructors
+        text: Construct; url: sec-construct
 urlPrefix: https://webidl.spec.whatwg.org/; spec: WEBIDL
     type: interface
         text: Promise; url: idl-promise
         text: nullable; url: idl-nullable-type
+    type: dfn
+        text: transfer array buffer; url: arraybuffer-transfer
```

Comment:
This should not be necessary if you use `[=ArrayBuffer/transfer=]` syntax.

Reply:
Sounds good. Fixed it in the latest commit.